webdriver/tests/bidi/input/perform_actions/wheel.py

```diff
@@ -87,12 +87,11 @@ async def test_scroll_iframe(
         actions=actions, context=top_context["context"]
     )
 
-    # Chrome requires some time to process the event from the iframe, so we wait for it.
+    # Chrome requires some time (~10-20ms) to process the event from the iframe, so we wait for it.
     async def wait_for_events(_):
         return len(await get_events(bidi_session, top_context["context"])) > 0
 
-    wait = AsyncPoll(bidi_session, timeout=0.5)
-    await wait.until(wait_for_events)
+    await AsyncPoll(bidi_session, timeout=0.5, interval=0.01, message='No wheel events found').until(wait_for_events)
```

Comment:
Just a nit. Feel free to ignore...

```suggestion
    await AsyncPoll(bidi_session, timeout=0.5, interval=0.01, message='No wheel events emitted').until(wait_for_events)
```